DEVNOTES.md

```diff
@@ -28,25 +27,25 @@ PUBLIC_URL='./'
 If you want to change these parameters you can build application by using the following command:
 
 ```sh
-npx cross-env API_PATH='{your_api_path_here}' PUBLIC_URL='{your_public_url_here}' yarn build
+npx cross-env API_PATH='{your_api_path_here}' PUBLIC_URL='{your_public_url_here}' npm build
```

Comment:
hm, as far as I understand we can use `npm build` because we still use npm cli 6.14.17
https://docs.npmjs.com/cli/v6/commands/npm-build

But I can update it to `npm run build` with no problem

+ converted PR to Draft not to accidentally merge it before we move to NPM